definitions/checks/check_sha1_certificate_authority.rb

```diff
@@ -0,0 +1,24 @@
+class Checks::CheckSha1CertificateAuthority < ForemanMaintain::Check
+  metadata do
+    label :check_sha1_certificate_authority
+    description 'Check if server certificate authority is sha1 signed'
+
+    confine do
+      feature(:katello)
+    end
+
+    do_not_whitelist
+  end
+
+  def run
+    installer_answers = feature(:installer).answers
+    server_ca = installer_answers[:certs][:server_ca_cert]
+
+    certificate = OpenSSL::X509::Certificate.new(File.read(server_ca))
+
+    assert(
+      certificate.signature_algorithm != 'sha1WithRSAEncryption',
+      "Server CA certificate signed with sha1 which will break on upgrade."
```

Comment:
Ok, and what shall I (as a user) do?

(read: can we make the printed error somehow actionable?)